.gitignore

```diff
@@ -10,6 +10,8 @@ build/
 artifacts/
 src/**/obj/
 src/**/bin/
+buildscripts/**/obj/
+buildscripts/**/bin/
```

Comment:
Interesting, so it sort of works like another project rather than the old solutions items.